### DISCUSSION TITLE
雷索纳斯市场讨论

### DISCUSSION THREAD

**Discussion by yjl9903** [yjl9903/resonance-market#7] [General]
https://resonance.breadio.wiki/

**Comment by yjl9903** [comment]
🎉🎉🎉 **雷索纳斯市场** 已正式接入 **[索思学会](https://github.com/kmou424/resonance-resodata)** 数据，绝大部分数据已经可以不用大家上传啦。

如有缺失数据仍然欢迎各位列车长帮忙补充，帮助 **雷索纳斯市场** 更好地服务各位列车长。

另外最近 **雷索纳斯市场** 正在征集新的名字，各位列车长如有兴趣可以移步 #39 提提意见。

在线使用：https://resonance.breadio.wiki/

GitHub：https://github.com/yjl9903/resonance-market/

问题反馈：https://github.com/yjl9903/resonance-market/issues/new

价格等数据问题反馈：https://github.com/yjl9903/resonance-market/issues/15

---

## 更新

- **03-06 21:05**：上报价格和价位验证
- **03-06 21:50**：展开的详情面板添加单票利润
- **03-07 03:00**：批量上报买入和卖出数据
- **03-07 16:30**：添加碳纤维和修复部分错误的基准价格
- **03-07 20:00**：之前免费的数据库被打爆了，更换了数据库，之前的日志应该丢失了
- **03-08 00:55**：重新实现了分布式缓存，如果发现出现很久的过期数据，麻烦及时反馈
- **03-11 15:55**：支持货物出售城市数据按利润排序 ( Thanks @Ximu-Luya )
- **03-13 01:52**：支持单元格详情弹窗快捷上报数据 ( @Ximu-Luya )
- **03-14 22:20**：雷索纳斯市场已正式自动化接入索思学会提供的情报数据 ( @Ximu-Luya )
- **03-16 00:38**：自动化数据更改为索思学会提供数据中的价格更新时间 ( @Ximu-Luya )
- **03-18 02:37**：支持按单票利润排序售出城市；利润计算规则修改为：售出价-买入价（按8%税率计算抬价砍价20%）( @Ximu-Luya )
- **03-19 02:09**：支持数据看板的城市筛选器( @Ximu-Luya )
- **03-25 02:09**：新增路线规划页面，按城市全特产商品单票总收益排序( @Ximu-Luya )

**Reply by DosiyoHu** [reply]
直接打开不会显示更新，只有上报过一次数据后才显示最新数据

**Reply by yjl9903** [reply]
@DosiyoHu 刚才检查了一下 cloudflare 的缓存配置，不知道现在是否有改善。

我自己感觉好像已经没什么问题了?

**Reply by GG20006** [reply]
我这边看没什么问题了，没上报也可以看到最新的

**Reply by theflybirds** [reply]
还有问题，不登录的话还是不更新，登录之后就好了

**Reply by jiguangyouling** [reply]
草，为啥我登录还是显示几小时前，是没人来更新了吗

**Reply by SetsunaZSeiei** [reply]
草哦，我这里也是只有上报后才会更新

**Reply by theflybirds** [reply]
现在也不行了 登录也不行 依然有问题

**Reply by dlmm10086** [reply]
我这里也是这问题

**Reply by yjl9903** [reply]
又清空了缓存，现在再试试呢？

**Reply by Nocumin** [reply]
现在好了

**Reply by theflybirds** [reply]
又不行了  啥情况 一会好一会不好

**Reply by yjl9903** [reply]
现在应该真的完全修好了

**Comment by YSJunagi** [comment]
给大佬点赞！

**Comment by FKzlw** [comment]
给大佬点赞！

**Comment by IridescentABX** [comment]
好好好

**Comment by ewzaoei** [comment]
### **金线快涨!**

**Comment by masterxiaxin** [comment]
帅

**Comment by GG20006** [comment]
加油加油！

**Comment by zmfk** [comment]
好耶！

**Comment by birdgao315** [comment]
怎么看最高收益？

**Reply by birdgao315** [reply]
明白了，绿色字体加粗的

**Reply by yjl9903** [reply]
会按照实时利润，给字体分配颜色，越深的利润越高。

之后考虑：单独弄一个图标标注最高利润地点，或者单独弄一列显示最佳目标地

**Reply by Pudding27sLee** [reply]
图标好些

**Reply by Ximu-Luya** [reply]
可以看看这个issue里的展示效果怎么样，售出城市会按利润从高到低依次排序，并忽略已失效的报价

https://github.com/yjl9903/resonance-market/issues/25

**Comment by suncloudrain** [comment]
支持

**Comment by cwan5232** [comment]
辛苦啦！感谢感谢

**Comment by PoLieber** [comment]
满仓！满仓！

**Comment by YSJunagi** [comment]
坏了，昨晚买的金沙搓成线到现在卖不出去（

**Reply by cwan5232** [reply]
我也

**Comment by 18975976110** [comment]
为啥我看的都是两小时前的昂

**Reply by xuanwu666** [reply]
我电脑端也是这样，但手机端有在更新

**Reply by 18975976110** [reply]
我也是手机端……看不了

**Reply by yjl9903** [reply]
为了优化访问速度和数据库资源消耗（用的免费的），更新了一些缓存机制，可能造成数据延迟。

先下掉缓存了，刷新看看呢。

**Reply by YSJunagi** [reply]
我看金线还是三小时前（
正常吗

**Reply by yjl9903** [reply]
<img width="1251" alt="image" src="https://github.com/yjl9903/resonance-market/assets/30072175/8124b673-4f2e-4d6f-bb2a-eaf0c3423eae">

**Reply by xuanwu666** [reply]
现在看了一下已经有很多数据是十几分钟以内的了，应该试好了

**Reply by YSJunagi** [reply]
嗷，原来是没人更新（

**Comment by Ambatur** [comment]
旧表刚没掉所以还需要互相扩散新表吧

**Reply by Ambatur** [reply]
能不能改一下设定啊，现在每个人发消息都会给我发邮件

**Reply by yjl9903** [reply]
这个好像不是我能控制的？

<img width="1344" alt="image" src="https://github.com/yjl9903/resonance-market/assets/30072175/f491ba85-98dc-4a32-b304-e9c4e1ba398c">

把当前 GitHub 仓库的通知关一关应该会好一点？

- https://github.com/yjl9903/resonance-market/discussions/7
- https://github.com/yjl9903/resonance-market

**Comment by shenqi998** [comment]
现在是没人更新吗？为什么我在都过期？

**Comment by zhang-wan-zhi** [comment]
我这一直是几个小时前的数据

**Reply by zmfk** [reply]
上传一个你所在的地区的数据刷新一下就好了

**Comment by Bing2048** [comment]
能不能加个一本书的利润呀？

**Comment by shikozh** [comment]
大佬可以加一项单批次的利润吗，对来回倒货、控制进货书用量很有帮助

**Comment by starnightlodging** [comment]
佬刚刚在荒原站孔雀石买入价格为1749，106%上升，但在这里无法上报（偏差过大）。

**Reply by starnightlodging** [reply]
现在为1683，102%下降也无法上报

**Reply by yjl9903** [reply]
荒原站孔雀石的基础价格不是 1337，106% 不是 1363？

是我的理解有问题吗

**Reply by starnightlodging** [reply]
孔雀石成本我这里显示的成本是1484，请问一下能上传图片吗？

**Reply by yjl9903** [reply]
<img width="705" alt="image" src="https://github.com/yjl9903/resonance-market/assets/30072175/c642acd0-09a9-4cb3-a118-3603d25c2fbf">

**Reply by starnightlodging** [reply]
往你邮箱（xlor@onekuma.cn）私信了下关于这个问题的图片附件

**Comment by HonTuT** [comment]
很多商品在铁盟哨站的实际价格、百分比与表格的基准涨跌不一致哇，炮弹、金箔、石墨烯之类的都没法更新游戏实际价格

**Reply by yjl9903** [reply]
我是从老黑的表格里全商品统计里爬的 https://www.kdocs.cn/l/cuPZppRuRJjO

统一到这个 issues 里讨论吧 https://github.com/yjl9903/resonance-market/issues/15

发一下正确的基准价格？

@HonTuT @starnightlodging

**Reply by starnightlodging** [reply]
https://docs.qq.com/sheet/DQlZhS1RQcHNranRU?u=76e178d9d96644649dfd6f933bf0c16d&tab=000002

**Comment by voidprohet** [comment]
刚刚试了几次，还是会出现直接打开不会显示更新，只有上报过一次数据后才显示最新数据的状态，并且刷新网页之后又是未更新的失效状态。

**Comment by Tachward** [comment]
登陆了也不更新啊

**Reply by Tachward** [reply]
上报一次价格就能看到更新了

**Comment by PigRider07** [comment]
先申个号看看情况，我在登陆之前是不更新的

**Comment by linlinlin0526** [comment]
我这里还是上报才能看到最新结果

**Comment by a978329926** [comment]
上报才能看到最新结果

**Comment by chaikatorabanto** [comment]
登陆了也还是需要上报一次数据才能看到最新价格 否则都是两三小时前的

**Comment by YUANSHEN2013** [comment]
还是上报后才会刷新

**Comment by keaibaobaoren** [comment]
卖个萌，喵~

**Comment by 9Pomika** [comment]
使用的大家要多多上报哟，方便每一个人呢

**Comment by PryceGe** [comment]
能不能用浅色色块的方式来突出那些利润较高的栏目？这样看上去应该会更加直观清晰一点。还有就是能不能把上报的功能整合到每一个栏目上，比如点淘金乐园-沙金（到修格里城）就能直接改价格，而不是到最后去按上报。不过这只是两个小意见，感谢帮了我很大的忙！

**Comment by tianxinshihuai** [comment]
战备工厂少个特产   碳纤维

**Comment by wuhulicheng** [comment]
????怎么数据空了

**Comment by Rlunpika** [comment]
只有我的利润错误吗？卖出地实时利润 - 购入地实时利润 ≠ 单位利润
还是说我理解有问题？T_T

**Reply by Bing2048** [reply]
议价了吧

**Reply by Rlunpika** [reply]
不太可能，比如7号港发财树23:47:09价格是5230，淘金乐园23:43:23是5734，不计算税率毛利是504，页面显示单位利润1152
议价5734*1.2 - 5230*0.8 = 2,696.8利润对不上，或者单计算砍价&抬价数值也不对

**Reply by Rlunpika** [reply]
5734x1.2 - 5230x0.8

**Reply by yjl9903** [reply]
直接用的老黑表格里的算法

![image](https://github.com/yjl9903/resonance-market/assets/30072175/bd4d7593-1056-4365-ac7e-4ee90aaaf451)

> P.S. 发现好像和之前不一样，所以又更新了一下

**Comment by YFJJj** [comment]
半夜金线涨到了120%往上，但是我等睡着了，啥也没卖😓

**Reply by Ximu-Luya** [reply]
我没继续赌，能源所金线115的时候卖了400个，赚了350个w

**Comment by VanNEW** [comment]
目前棉花向自由港卖利润有600多了，不考虑加一条吗

**Comment by a190141074** [comment]
能源所的金线快涨啊，悲，屯着500+只能干耗着

**Reply by a190141074** [reply]
好好好，线材115了

**Comment by lengxining** [comment]
游戏机这算的有问题啊 基础3200 售价3776 显示利润1062

**Reply by lengxining** [reply]
原地还涨了 106

**Comment by Ximu-Luya** [comment]
开始屯显卡了，显卡80了

**Reply by Ximu-Luya** [reply]
曼德显卡80，自由港120，开倒！

**Reply by Ximu-Luya** [reply]
悲，自由港显卡降了，就倒了两趟赚了300w

**Comment by YUANSHEN2013** [comment]
这不对吧，7600-7500=1600的利润？？？

**Reply by YUANSHEN2013** [reply]
发电机的利润计算有大问题

**Reply by starnightlodging** [reply]
你指发电机哪里到哪里的利润

**Reply by Ximu-Luya** [reply]
我看了一下，到淘金的确实是这样。不过这个计算规则暂时是从老黑表格里抄过来的，后续我们考虑优化一下，自己设计一套计算规则吧。

**Comment by shadowlisp** [comment]
![起飞](https://github.com/shadowlisp/image/blob/main/5K%24R6PT%40BE7%7D3%5BDG6%5D%602MNX.png)

**Comment by XWgy** [comment]
上报选错城市怎么改回去啊？

**Reply by YSJunagi** [reply]
点进你上传错误信息对应格子的历史记录然后删除

**Comment by NightVerulia** [comment]
铁盟哨站的炮弹基础百分比例不对

**Reply by NightVerulia** [reply]
比如炮弹在游戏里的价格是xx，显示82↑，但我把xx填进去，表格显示的却是84↑，应该是炮弹原始的价格出错了

**Comment by dgklr** [comment]
打算写一个综合利润最佳的程序（指买入几个不同的东西然后计算哪里利润最佳）
毕竟如果只有单一价格排序显得不是特别够用）

**Reply by Ximu-Luya** [reply]
如果可以的话，欢迎贡献代码

**Comment by emeiyan** [comment]
佬，能不能搞一个切换单位利润和单票利润显示的按钮啊，倒货的时候有的东西看起来卖的贵实际上就那么一两个，只能点进去看单票利润

**Reply by Ximu-Luya** [reply]
是用的手机端查看的数据吗？你这里指的是得点一下才能在额外展示的面板中看到单票利润吗

**Reply by emeiyan** [reply]
是的，现在的话想知道哪个单票利润高还得一个一个点进去看，然后对比。想问能不能搞个按钮直接把首页显示的单位利润切换成单票利润，就不用点进去一个一个看了

**Reply by Ximu-Luya** [reply]
可以，今晚我看看就做吧

**Comment by LeiWuHen666** [comment]
大家积极更新数据啊

**Reply by Ximu-Luya** [reply]
雷索纳斯市场已经正式接入自动化数据啦，大家如果等级到了40级，欢迎帮忙补充一下 `阿妮塔战备工厂` 的数据

**Comment by tianxinshihuai** [comment]
石墨烯-曼德矿场 数据区间不对，90%的价格为4286

**Reply by tianxinshihuai** [reply]
不是不是，我看错了，不用改

**Comment by tianxinshihuai** [comment]
炮弹——曼德矿场 数据区间不对，95%价格1673

**Comment by yfTsssss** [comment]
可以给按利润排序设置个url链接（后加参数什么的）吗，想直接打开就按利润排不用再点一下

**Reply by Ximu-Luya** [reply]
我们可能不会提供利润排序的url链接，不过后续可以这个设置默认记录在本地，每次打开就不用再额外选了。感谢你提出的建议，已经记录在issue里了 #41，本周应该就会完成。

**Comment by duboas** [comment]
希望在数据失效的时候基准价格不会消失

**Reply by Ximu-Luya** [reply]
收到了喔，已建立issue #43

**Comment by XxiaoxiaoX** [comment]
希望能加入一批货去每个站的总和收益，再根据所有的总和收益算出最佳往返路线就更好了

**Reply by LeiWuHen666** [reply]
那就相当于帮你玩游戏了

**Reply by Ximu-Luya** [reply]
这个功能已经在计划中辣，不过当前进度是“新建文件夹中……”咕咕咕，已经有挺多优化、bug的todolist了。这个是下一个大的新功能，敬请期待吧

**Comment by bw0963** [comment]
之前经常出现连不上的问题 现在基本不会了 是优化好了？（/赞）

**Reply by Ximu-Luya** [reply]
很尴尬，并没有做什么。本来计划弄个镜像站的，遇到问题搁置了，或许单纯是因为你那儿的网络OK了。也有可能后端大佬调了一下cf的节点？

**Comment by YYBYX** [comment]
这个利润计算为什么那么怪？比如就现在的实时价格：发动机修城2892，哨站3949，利润不应该是1057么，但这个上面利润却是1624。。。

**Reply by Ximu-Luya** [reply]
利润计算规则是从老黑表中迁移过来的，大致计算了买时砍价和卖时抬价，所以就算买入价格比卖出价格高时也有可能会是正利润。不过不用担心，这部分我已经在优化了，到时候默认会展示基准利润（也就是不计算砍价抬价和税），同时也支持设置为全计算利润，请列车长稍等几天🥺

**Reply by tianxinshihuai** [reply]
这个不是抬砍价的问题，有一个额外的利润数值和距离有关，和原产地距离越远价格越高，这就是为什么同样的金线研究所117高于7号120的原因

**Comment by duboas** [comment]
在没有右侧上报数据按钮，如果数据全部删除不能弹出快速上报的弹窗

**Reply by Ximu-Luya** [reply]
这种特殊情况，列车长可以通过顶部导航栏中的“上报价格”中，选择相应信息进行价格上报。当前部分商品的部分城市交易基准数据缺失，导致无法自动上报，后续会修复好此问题，列车长们就可以不用再手动上报啦~

**Comment by sandan20001116** [comment]
是不是商品是不全的？比如说七号自由港只有5种商品，其中还有个礼包还是自己做的

**Reply by Ximu-Luya** [reply]
因为目前暂时只展示了特产商品

**Reply by sandan20001116** [reply]
好的，谢谢大佬

**Comment by LukeLawliet** [comment]
我穿越了，更新日志都更到3月29日了！

**Comment by Kopronimus** [comment]
数字必须是大于70%。

**Reply by Ximu-Luya** [reply]
已修复。不过现在所有商品数据均已支持自动化上报，部分商品在交易量低，在间隔1小时才更新新的价格导致可能看到部分数据展示失效，请列车长稍等片刻，数据就正常同步过来了

**Comment by Ano-Mowen** [comment]
感谢大佬分享，最近倒货发现有货物价格与实际不符（铁萌站货物价格浮动超过15%），不排除有个别玩家篡改价格的可能性。这边只是怀疑，因为图表信息有滞后性无法确定。

**Reply by Ximu-Luya** [reply]
请问有具体的例子吗？铁盟哨站到哪个城市的哪个货物，我看看情况

**Reply by Ano-Mowen** [reply]
我记得好像是铁萌到阿尼塔战备，然后时间大概是23号早上5点+-1h。

**Comment by Winterllling** [comment]
还是建议手动上报加点限制，真的感觉有人不怀好意。刚才能源120%的金线不知道谁改成97%，后面看到自动上报是115%才发现价格不对，我屯的这批金线因为这个直接少5% 利润了

**Reply by Ximu-Luya** [reply]
已记录issue #62

**Reply by Alex-c-yl** [reply]
从凌晨开始很多数据就没有再更新，昨晚跑商时我手动更新了好几次7号自由港到曼德矿场的数据，自动化数据是停摆了吗？

**Reply by Ximu-Luya** [reply]
@Alex-c-yl 已修复

**Comment by Qiiamx** [comment]
今天空了跑商，发现新增了规划，遂好奇规划的进度条中，100%代表什么，简单看了下好像是假的100%哈哈哈
我能知道某条路线的“单票利润上限”与“当前单票利润”的差距吗（或者上限排序？

**Reply by Ximu-Luya** [reply]
这并不是什么进度条哈哈，仅仅是用来一眼查看当前路线与最高收益路线相比的情况，最高收益路线占比90%，最低收益路线占比0%

当前路线与最低收益的路线的差值占最高收益路线与最低收益路线的差值，即这个“进度条”长度

**Reply by Ximu-Luya** [reply]
> 某条路线的“单票利润上限”与“当前单票利润”的差距

@Qiiamx 个人认为，就算直到单票利润上限，对列车长们的帮助也意义不大。假设有一条路线已经达到或者无限接近了其单票利润上限，即所有商品都接近120%，但其就算达到上限也完全比不过其他路线，那这种计算其实有点无意义。

不过我觉得，你说的仅从一条路线的维度上来说，查看当前路线的当前收益和上限收益的差距还是有意义的，我会根据实际情况积极参考你的意见，非常感谢你提供的建议！

**Reply by Qiiamx** [reply]
感谢你的回复

这样看来我确实已经不会用自然语言描述功能了，总之是个进度条组件（
正如你所说，最高利润的具体值对时间自由的玩家意义不大，所以我觉得这个想法还有不少改进空间
最明显的问题就是，当等级解锁城市多了的时候， 低等级玩家的理论上限肯定是不一样的

__所以单程/双程 利润的历史记录会不会好？__

毕竟我只是希望书花在想对更赚的时间，不需要参考货物有哪些，这样或那样的个性化配置，有一个客观的“显卡天梯”就可以了

这个新想法来自本站的商品历史价格（助我找时间摸鱼

**Comment by oOtroyOo** [comment]
设置规则没有缓存吗，每次刷新就要再点一下 🤨

**Reply by Ximu-Luya** [reply]
@oOtroyOo 嗯对的，暂时还没有，下一个要做的就是设置页改造，还请等一两天

**Comment by oOtroyOo** [comment]
🤨 大家伙有没有什么办法，接个RSS或者HOOKS或者什么脚本，订阅个价格区间，能发出通知，老盯着也不是个办法。 本来想自己搞个脚本，但瞄了一眼索思学会 数据，get api 好像也要key，我以为post才要key。

**Comment by lalgodlike** [comment]
什么时候把发射中心加上

**Reply by Qiiamx** [reply]
感觉发射中心跟澄明一样鸡肋，打算直接体力刷龙，换了ur走人了

**Reply by Ximu-Luya** [reply]
@lalgodlike 预计5.4-5.5，现在人不在家里

**Reply by yjl9903** [reply]
加上了

**Comment by oOtroyOo** [comment]
怎么经常
Error 1101 
Worker threw exception

**Reply by yjl9903** [reply]
首页吗，有截图吗？

你的网络环境呢，最好挂个梯子？

**Reply by Ximu-Luya** [reply]
@yjl9903 我也遇到过，是cf的提示。我估计是服务端异常导致的，上报脚本在上报数据的时候有时候也会报错500，具体怎么引起了还不太清楚。

**Reply by yjl9903** [reply]
在测试环境能复现，我看看

应该是前天上的 server-sent events

**Comment by acseradialex** [comment]
这两天是不是出问题了

**Reply by siuzuya** [reply]
会不会后台出问题了？隔壁科伦巴商会也是数据不准确

**Reply by Ximu-Luya** [reply]
@acseradialex @siuzuya 数据已恢复正常，感谢反馈

**Comment by Alex-c-yl** [comment]
纯金线材在发射中心的基准价格有误，109% 售价8275，实际基准价格应该在7590左右

**Reply by Alex-c-yl** [reply]
玛瑙104%售价2976，实际基准价格在2860左右，青金石112%3562,基准价格3180

**Reply by Alex-c-yl** [reply]
哦，我明白了，发射中心基准价格现在是直接映射了战备中心的……发射中心实际上比能源研究所还高一档，数字纠正过来不出意外都是路线规划前排常客

**Reply by Ximu-Luya** [reply]
@Alex-c-yl 非常感谢您的反馈，您提到的问题阿妮塔观测枢已确定，正在向索思学会确认数据问题，在有下一步结论时，阿妮塔观测枢的开发人员会在这里向你同步最新情况，再次感谢您向我们反馈。

**Reply by Ximu-Luya** [reply]
@Alex-c-yl 已与索思学会确认基准数据未更新，但索思学会代表暂时无力调整，目前我仅发现了淘金乐园商品卖到发射中心存在基准价格异常，预计这两天内有时间会修复。如果列车长还发现其他异常数据，可以在这里向我反馈，也可以去Github仓库提Issue

**Comment by Vavilov3301** [comment]
您好，数据更新似乎出现了问题，目前的数据均停留在大约13日11：30左右，全页面数据均为灰色。
非常感谢开发者和创作页面的团队的努力以及提供的便利！非常感谢你们的页面，真的在我的行车之旅中帮了大忙
祝你们一天顺利

**Reply by Ximu-Luya** [reply]
@Vavilov3301 感谢您的报告，数据已恢复正常

**Reply by Vavilov3301** [reply]
页面已经恢复正常了，非常感谢更新和修复
再次你们提供的便利和数据

祝一天愉快

**Reply by Vavilov3301** [reply]
很抱歉再次打扰

目前全页数据大部分已经变成空白，剩下的数据均为灰色

祝一天顺利

**Comment by Vavilov3301** [comment]
很抱歉再次打扰

目前全页数据大部分已经变成空白，剩下的数据均为灰色
谢谢你们的努力

祝一天顺利

**Reply by Vavilov3301** [reply]
页面加载恢复了，非常感谢你们提供的数据支持。不仅准确，更新也非常快，祝你们一直好运

**Reply by Ximu-Luya** [reply]
非常感谢列车长的反馈

**Comment by Koumloasde** [comment]
您好，
数据似乎又出现了异常，目前全页数据大部分已经变成空白，可否早日恢复？
祝一天顺利。

**Reply by Ximu-Luya** [reply]
已恢复，感谢反馈。

**Comment by D0ngfang** [comment]
很抱歉打扰

目前全页数据一部分已经变成空白，剩下的数据均为灰色
谢谢你们的努力

祝一天顺利

**Reply by Ximu-Luya** [reply]
@D0ngfang 数据已恢复，感谢反馈

**Reply by D0ngfang** [reply]
页面加载恢复了，非常感谢你们提供的数据支持。为跑商提供了很大帮助

**Comment by winerchan** [comment]
您好，抱歉打扰您了
数据似乎停留在了昨天16:50:23
昨晚我个人上报了几个站点的数据，但是今天依然没有自动更新，能否麻烦看一下
非常感谢各位的付出

**Reply by Ximu-Luya** [reply]
@winerchan 感谢列车长的反馈，数据已恢复

**Comment by Aland689** [comment]
9月2日，所有市场数据全部空白，并在今天我上传了修格里城和铁盟哨站哨站的价格信息，请问这个网站已经被废弃了吗？

**Reply by Ximu-Luya** [reply]
@Lijia2237 感谢反馈，所有数据均已更新。确实现在已处于有限维护状态，数据大批量失效时还请反馈，我这边会及时处理。

**Comment by yunjinanana** [comment]
目前整个首页表格为空，路线规划数据却是正常，能请问使出了什么问题吗，先感谢大佬的奉献了，退坑一年回归还是熟悉的网站，让人怀念又安心。

**Reply by Ximu-Luya** [reply]
很可惜，此项目目前已经停止维护了。服务器仍然在正常允许，作为玩家和用户还是可以上报价格数据，全城市数据自动上报的功能已经下掉了，并且游戏中新开放的城市也并没有添加，抱歉。

**Comment by yunjinanana** [comment]
好的，感谢您们以前的辛勤付出！雷索纳斯市场这个网站对刚玩这个游戏的我确实是带来了莫大的帮助，也让我真正体会到了跑商的乐趣，最后祝您们未来生活美满，阖家欢乐！(◦˙▽˙◦)



---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2025年08月07日 14:25 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [yjl9903/resonance-market] 雷索纳斯市场讨论 (Discussion #7) |

很可惜，此项目目前已经停止维护了。服务器仍然在正常允许，作为玩家和用户还是可以上报价格数据，全城市数据自动上报的功能已经下掉了，并且游戏中新开放的城市也并没有添加，抱歉。
